src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityEnroll.jsx

```diff
@@ -150,6 +166,8 @@ class AccountSecurityEnroll extends AsyncView {
       // Otherwise API will think that we are on verification step (e.g. after submitting phone)
       otp: hasSentCode ? this._form.otp || '' : undefined,
       secret: authenticator.secret,
+      ...(this.memberId && {memberId: this.memberId}),
```

Comment:
Can we combine this into a single object e.g. `this.foo = {}`

Then add in memberId/token as needed and you can always spread `this.foo`